Marlin/Configuration_adv.h

```diff
@@ -597,6 +597,8 @@
   //#define DOUBLECLICK_FOR_Z_BABYSTEPPING // Double-click on the Status Screen for Z Babystepping.
   #define DOUBLECLICK_MAX_INTERVAL 1250 // Maximum interval between clicks, in milliseconds.
                                         // Note: Extra time may be added to mitigate controller latency.
+  //#define BABYSTEP_ZPROBE_GFX_OVERLAY // Enable graphical overlay on Z-offset editor
```

Comment:
Need to add these new options to the example configuration files. Also, you should probably add a sanity check that BABYSTEP_ZPROBE_GFX_OVERLAY is only defined if DOGLCD is defined, since this won't work with the 20x4 LCDs.